qt/widgets/common/src/QtPropertyBrowser/qteditorfactory.cpp

```diff
@@ -203,9 +136,9 @@ void QtSpinBoxFactoryPrivate::slotSetValue(int value) {
     Creates a factory with the given \a parent.
 */
 QtSpinBoxFactory::QtSpinBoxFactory(QObject *parent)
-    : QtAbstractEditorFactory<QtIntPropertyManager>(parent) {
+    : QtSpinBoxFactoryBase<QSpinBox>(parent) {
   d_ptr = new QtSpinBoxFactoryPrivate();
```

Comment:
This may not work within the structure here but would it be possible to have this be a unique pointer rather than a raw pointer which we delete in the destructor?

Reply:
So the Qt pointer to implementation pattern uses raw pointers throughout, I don't think it would be trivial (or worthwhile at the moment to change it to smart pointers).